IAMOTZ/More-recipes
# #152082784 API for posting reviews

#### What should this PR do?
It should enable posting reviews for recipes in a dummy database.
#### Description of Task to be completed?
User can post reviews for recipes(API endpoint).
#### How should this be manually tested?
- clone the repo
- cd into the server directory
- run the following commands
- `npm install` to install dependencies
- `npm run start:dev` to start the server on port 3000
- On postman or any API testing tool, send post request to `/api/recipes/<recipeId>/reviews`
- The post request should be accompanied by a body containing: 
`{ reviews: ""}`
- Fix in whatever you like as the value of the reviews
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152082784
#### Screenshots (if appropriate)
None
#### Questions:
None
